config/packages/parameters.yml.dist

```diff
@@ -11,17 +11,16 @@ parameters:
         - en_GB
 
     # WebAuthn user display name
-    user_display_name: SURFsecureID
-    webauthn_name: SURFsecureID
+    user_display_name: SURFsecureID Dev Environment
```

Comment:
This is to become part of the OpenConext project, right? I'd call it OpenConext and use full caps for the environment. E.g.: `OpenConext-Stepup DEV environment`